src/modules/pua_json/pua_json_publish.c

```diff
@@ -0,0 +1,387 @@
+/*
+ * PUA_JSON module interface
+ *
+ * Copyright (C) 2010-2014 2600Hz
+ *
+ * This file is part of Kamailio, a free SIP server.
+ *
+ * Kamailio is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version
+ *
+ * Kamailio is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ * Contributor(s):
+ * Emmanuel Schmidbauer <eschmidbauer@gmail.com>
```

Comment:
@eschmidbauer sorry for this.. but, what % of this code is actually `only` yours ? same thing in other files on this `new` module. don't get me wrong, i approve this move from a `kazoo` centric solution to a generic one and i'm glad you found the time to make this happen.

Reply:
If you look at line 4 of the files that were mostly borrowed from kazoo, i included:
``` * Copyright (C) 2010-2014 2600Hz ```